src/control/ras/event_test.go

```diff
@@ -0,0 +1,24 @@
+//
```

Comment:
Did you mean to add some content to this file?

Reply:
yes placeholder